tests/azure-sdk-for-cpp/Makefile

```diff
@@ -0,0 +1,46 @@
+TOP=$(abspath ../..)
+include $(TOP)/defs.mak
+
+ifdef STRACE
+OPTS = --strace
+endif
+
+APPDIR=$(CURDIR)/appdir
+APPBUILDER=$(TOP)/scripts/appbuilder
+
+ROOTFS=$(CURDIR)/rootfs
+
+export TIMEOUT=10000
```

Comment:
Can you put comments on where the timeout is used?

Reply:
The timeout is not used in solutions, it is used if this test were to be run in the test pipeline in the future. Keeping it here as an indication of how much time the test should take